src/client/dfuse/dfuse.h

```diff
@@ -19,8 +19,7 @@
 
 #include <daos.h>
 #include <daos_fs.h>
-
-#include "dfs_internal.h"
+#include <daos/dfs_lib_int.h>
```

Comment:
Minor but I think these should probably be using quotes still.  We have plenty of code that does both though.
```suggestion
#include "daos/dfs_lib_int.h"
```